libs/stanag4609/src/main/java/org/codice/alliance/libs/stanag4609/PESUtilities.java

```diff
@@ -30,15 +33,16 @@
 
     private static final int PRIVATE_STREAM_ID = 0xBD;
 
+    private static final Set<Integer> KLV_STREAMS = new HashSet<>(Arrays.asList(METADATA_STREAM_ID,
+            PRIVATE_STREAM_ID));
+
     public static DecodedKLVMetadataPacket handlePESPacketBytes(final byte[] pesPacketBytes,
             KlvDecoder decoder) throws KlvDecodingException {
         final MPSDemuxer.PESPacket pesHeader =
                 MPSUtils.readPESHeader(ByteBuffer.wrap(pesPacketBytes), 0);
 
-        if (pesHeader.streamId == METADATA_STREAM_ID) {
-            return new SynchronousMetadataPacket(pesPacketBytes, pesHeader, decoder).decodeKLV();
-        } else if (pesHeader.streamId == PRIVATE_STREAM_ID) {
-            return new AsynchronousMetadataPacket(pesPacketBytes, pesHeader, decoder).decodeKLV();
+        if (KLV_STREAMS.contains(pesHeader.streamId)) {
+            return new MetadataPacket(pesPacketBytes, pesHeader, decoder).decodeKLV();
```

Comment:
We still need a way to handle Metadata Access Units for synchronous metadata streams.